tasks/section4.yml

```diff
@@ -71,244 +89,257 @@
+      src: audit/rhel7cis_rule_4_1_6.rules.j2
+      dest: /etc/audit/rules.d/rhel7cis_rule_4_1_6.rules
+      owner: root
+      group: root
+      mode: 0600
   when:
       - rhel7cis_rule_4_1_6
+  notify: restart auditd
   tags:
       - level2
       - auditd
       - patch
       - rule_4.1.6
-      - notimplemented
 
 - name: "SCORED | 4.1.7 | PATCH | Ensure events that modify the system's Mandatory Access Controls are collected"
-  command: /bin/true
-  changed_when: no
+  template:
+      src: audit/rhel7cis_rule_4_1_7.rules.j2
+      dest: /etc/audit/rules.d/rhel7cis_rule_4_1_7.rules
+      owner: root
+      group: root
+      mode: 0600
   when:
       - rhel7cis_rule_4_1_7
+  notify: restart auditd
   tags:
       - level2
       - auditd
       - patch
       - rule_4.1.7
-      - notimplemented
 
 - name: "SCORED | 4.1.8 | PATCH | Ensure login and logout events are collected"
-  command: /bin/true
-  changed_when: no
+  template:
+      src: audit/rhel7cis_rule_4_1_8.rules.j2
+      dest: /etc/audit/rules.d/rhel7cis_rule_4_1_8.rules
+      owner: root
+      group: root
+      mode: 0600
   when:
       - rhel7cis_rule_4_1_8
+  notify: restart auditd
   tags:
       - level2
       - auditd
       - patch
       - rule_4.1.8
-      - notimplemented
 
 - name: "SCORED | 4.1.9 | PATCH | Ensure session initiation information is collected"
-  command: /bin/true
-  changed_when: no
+  template:
+      src: audit/rhel7cis_rule_4_1_9.rules.j2
+      dest: /etc/audit/rules.d/rhel7cis_rule_4_1_9.rules
+      owner: root
+      group: root
+      mode: 0600
   when:
       - rhel7cis_rule_4_1_9
+  notify: restart auditd
   tags:
       - level2
       - auditd
       - patch
       - rule_4.1.9
-      - notimplemented
 
 - name: "SCORED | 4.1.10 | PATCH | Ensure discretionary access control permission modification events are collected"
-  command: /bin/true
-  changed_when: no
+  template:
+      src: audit/rhel7cis_rule_4_1_10.rules.j2
+      dest: /etc/audit/rules.d/rhel7cis_rule_4_1_10.rules
+      owner: root
+      group: root
+      mode: 0600
   when:
       - rhel7cis_rule_4_1_10
+  notify: restart auditd
   tags:
       - level2
       - auditd
       - patch
       - rule_4.1.10
-      - notimplemented
 
 - name: "SCORED | 4.1.11 | PATCH | Ensure unsuccessful unauthorized file access attempts are collected"
-  command: /bin/true
-  changed_when: no
+  template:
+      src: audit/rhel7cis_rule_4_1_11.rules.j2
+      dest: /etc/audit/rules.d/rhel7cis_rule_4_1_11.rules
+      owner: root
+      group: root
+      mode: 0600
   when:
       - rhel7cis_rule_4_1_11
+  notify: restart auditd
   tags:
       - level2
       - auditd
       - patch
       - rule_4.1.11
-      - notimplemented
 
 - name: "SCORED | 4.1.12 | PATCH | Ensure use of privileged commands is collected"
-  command: /bin/true
+  shell: for i in  $(df | grep '^/dev' | awk '{ print $NF }'); do find $i -xdev -type f -perm -4000 -o -type f -perm -2000 2>/dev/null; done
```

Comment:
Should use `command` here if we can...although this might be one of those commands that has to be run with `shell`.

Reply:
Unfortunately this one can't be changed as command doesn't support piping